cernopendata_client/cli.py

```diff
@@ -125,7 +125,13 @@ def get_metadata(server, recid, doi, title, output_value):
     type=click.STRING,
     help="Which CERN Open Data server to query? [default={}]".format(SERVER_HTTP_URI),
 )
-def get_file_locations(server, recid, doi, title, protocol, expand):
+@click.option(
+    "--verbose",
+    is_flag=True,
+    default=False,
+    help="Output the file size and the checksum information.",
```

Comment:
Output also the file size (in the second column) and the file checksum (in the third column).